### PR TITLE
don't fail sendmail when all worker builds fail

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by rcerven** [comment]
maybe I can add there in that case 'name' label from Dockerfile.

thoughts ? @twaugh  @MartinBasti  @athos-ribeiro

**Comment by twaugh** [comment]
Yes, I'd rather we used the name label really. Ideally we'd include the git reference in that message, IMHO.

**Comment by rcerven** [comment]
using name label for image, and also added git url & ref

File: tests/plugins/test_sendmail.py [hunk 1]
```diff
@@ -490,28 +521,41 @@ class WF(object):
...
             'Status: ' + status,
             'Submitted by: ',
+            'Source url: ' + git_source_url,
+            'Source ref: ' + git_source_ref,
         ]
+        exp_body.extend(common_body)
+
         if autorebuild:
-            exp_body[-1] += '<autorebuild>'
+            exp_body[-3] += '<autorebuild>'
```
Comment (athos-ribeiro) [inline tests/plugins/test_sendmail.py]: It would be nice to refactor this to avoid using thise index here

File: tests/plugins/test_sendmail.py [hunk 2]
```diff
@@ -490,28 +521,41 @@ class WF(object):
...
+            exp_body[-3] += SendMailPlugin.DEFAULT_SUBMITTER
 
         if log_url_cases[(koji_integration, autorebuild, success)]:
             if has_koji_logs:
-                exp_body.append("Logs: https://koji/work/tasks/12345")
+                exp_body.insert(-2, "Logs: https://koji/work/tasks/12345")
             else:
-                exp_body.append("Logs: "
-                                "https://something.com/builds/blablabla/log")
+                exp_body.insert(-2, "Logs: https://something.com/builds/blablabla/log")
```
Comment (athos-ribeiro) [inline tests/plugins/test_sendmail.py]: and here

**Comment by rcerven** [comment]
rn updated: - don't fail sendmail plugin when all worker build fails

**Pull request merged**